rust/crd/src/lib.rs

```diff
@@ -169,17 +169,17 @@ pub struct AirflowClusterConfig {
     pub expose_config: Option<bool>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub load_examples: Option<bool>,
-    /// In the future this setting will control, which ListenerClass <https://docs.stackable.tech/home/stable/listener-operator/listenerclass.html>
-    /// will be used to expose the service.
-    /// Currently only a subset of the ListenerClasses are supported by choosing the type of the created Services
-    /// by looking at the ListenerClass name specified,
-    /// In a future release support for custom ListenerClasses will be introduced without a breaking change:
+    /// This enum controls which type of Service the Operator creates for this AirflowCluster:
```

Comment:
Enum is the rust/programming name, this is basically user facing doc, i would say `field` instead of `enum` ?

Reply:
I changed it!